src/test/java/id/ac/itb/logistik/ditlog/DitlogUserServiceTests.java

```diff
@@ -0,0 +1,148 @@
+package id.ac.itb.logistik.ditlog;
+
+import id.ac.itb.logistik.ditlog.model.User;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.*;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.xml.bind.DatatypeConverter;
+import java.security.MessageDigest;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class DitlogUserServiceTests {
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
```

Comment:
Change the testing to add its own test data to local db instead of using the production db